src/drivers/common/memory.h

```diff
@@ -11,15 +11,20 @@
 
 #include <stdint.h>
 
+#define MAX_SEGMENTS 64
```

Comment:
There is no need to define MAX_SEGMENTS in the header. You can move it to memory.c

Reply:
Fixed.